components/build/build-service/.tekton/tektoncd-results.yaml

```diff
@@ -0,0 +1,7 @@
+apiVersion: pipelinesascode.tekton.dev/v1alpha1
```

Comment:
should not be under build-service but under tekton-results, also the file should be included in kustomization.yaml

Reply:
Got it thanks